src/index.html

```diff
@@ -7,12 +7,70 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <section
+      class="product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__product-image"
+        src="./images/imac.jpeg"
+        alt="apple-imac-27"
+      />
+
+      <div class="product-card__product-details">
+        <h3 class="product-card__product-details-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+
+        <p class="product-card__product-details-description">
+          Product code: 195434
+        </p>
+      </div>
+
+      <div class="product-card__rating">
+        <div class="stars stars--4">
```

Comment:
The `div` element with the class `stars` should be implemented as a separate BEM block since it represents a reusable component. This will improve the scalability of the CSS and follow the BEM methodology more closely.